src/db/z_db.erl

```diff
@@ -861,9 +875,7 @@ equery1(DbDriver, C, Sql) ->
     equery1(DbDriver, C, Sql, [], ?TIMEOUT).
 
 equery1(DbDriver, C, Sql, Parameters) when is_list(Parameters); is_tuple(Parameters) ->
-    equery1(DbDriver, C, Sql, Parameters, ?TIMEOUT);
-equery1(DbDriver, C, Sql, Timeout) when is_integer(Timeout) ->
-    equery1(DbDriver, C, Sql, [], Timeout).
+    equery1(DbDriver, C, Sql, Parameters, ?TIMEOUT).
 
 equery1(DbDriver, C, Sql, Parameters, Timeout) ->
     case DbDriver:equery(C, Sql, Parameters, Timeout) of
```

Comment:
Remove the dynamic function call on line 881. Only modules that define callbacks should make dynamic calls.